infrastructure/game-client-startup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Script to install applications for the Game Client VM 
+
+export DEBIAN_FRONTEND=noninteractive
+
+curl -1sLf 'https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-l4t/setup.deb.sh' | sudo -E bash
```

Comment:
So with moonlight - on this machine we'll need the host, not the client (this is the client)

Here is an open source host:
https://github.com/LizardByte/Sunshine

You can also see this on the website https://moonlight-stream.org/#

![image](https://github.com/googleforgames/global-multiplayer-demo/assets/298370/5763f860-8ea9-46a7-9c1c-43bc12dcd61d)